src/views/ProductsContainersView.vue

```diff
@@ -168,8 +168,7 @@ onMounted(async () => {
   const vatGroupsResp = await apiService.vatGroups.getAllVatGroups();
   vatGroups.value = vatGroupsResp.data.records;
 
-  // TODO: Put getAllContainers into container store and take care of pagination
-  // See: https://github.com/GEWIS/sudosos-frontend-vue3/issues/51
+
```

Comment:
why was it removed?

Reply:
I thought it was not really an issue, but seeing that you already made a branch to fix it, I'll happily add it back.